docs/.vitepress/contributors.ts

```diff
@@ -99,10 +107,28 @@ const plainTeamMembers: CoreTeam[] = [
     avatar: contributorsAvatars.zxch3n,
     name: 'Zixuan Chen',
     github: 'zxch3n',
+    mastodon: 'https://elk.zone/hachyderm.io/@zx',
     twitter: 'zxch3n',
     title: 'A fullstack developer',
     desc: 'Creating tools for collaboration',
```

Comment:
My mastodon link is correct. Can you help me update this to “Working on CRDTs & local-first software”

Reply:
check latest changes

Reply:
Thanks, but I meant:

Title: A fullstack developer
Desc: Working on CRDTs & local-first software

Reply:
done